tests/foreman/api/test_errata.py

```diff
@@ -53,18 +51,17 @@ def activation_key(module_org, module_lce):
 
 
 @pytest.fixture(scope='module')
-def rh_repo(module_org, module_lce, module_cv, activation_key):
+def rh_repo(module_entitlement_manifest_org, module_lce, module_cv, activation_key):
```

Comment:
is this change intended?

Reply:
Yes, this is intended. In converting modules with local fixture stacks, I am generally replacing module_org with module_entitlement_manifest_org if the module_org previously had a manifest imported (as was the case here with the `force_manifest_upload=True` option).

Reply:
Yes, this is intended. For modules with local fixture stacks, I have been changing module_org to module_entitlement_manifest_org in cases such as this one where a manifest is uploaded to the org, which is done here with the `force_manifest_upload=True` option to `setup_org_for_a_rh_repo`.